chassis/core/src/main/java/com/griddynamics/jagger/util/generators/TestDescriptionGenerator.java

```diff
@@ -0,0 +1,33 @@
+package com.griddynamics.jagger.util.generators;
+
+import com.griddynamics.jagger.engine.e1.scenario.OneNodeCalibrator;
+import com.griddynamics.jagger.engine.e1.scenario.WorkloadTask;
+import com.griddynamics.jagger.invoker.QueryPoolScenarioFactory;
+import com.griddynamics.jagger.invoker.RoundRobinPairSupplierFactory;
+import com.griddynamics.jagger.invoker.SimpleCircularLoadBalancer;
+import com.griddynamics.jagger.user.test.configurations.JTestDescription;
+
+/**
+ * @author asokol
+ *         created 11/6/16
+ *         Generate {@link WorkloadTask} entity from user-defined {@link JTestDescription} entity.
+ */
+class TestDescriptionGenerator {
+
+    static WorkloadTask generatePrototype(JTestDescription jTestDescription) {
+        WorkloadTask prototype = new WorkloadTask();
+        prototype.setCalibrator(new OneNodeCalibrator());
```

Comment:
Why this value is hardcoded? We don't want to use another implementations of Calibrator?